Formula/radare2.rb

```diff
@@ -24,6 +24,7 @@ class Radare2 < Formula
   stable do
     url "http://cloud.radare.org/get/1.1.0/radare2-1.1.0.tar.gz"
     sha256 "7bc1e206a2b4def6bdb8684c2af0281b007986a0b5b5da652bd03be264ca0fa5"
+    revision 1
```

Comment:
This is in the wrong place. Applies to the whole formula, so needs to live outside any `stable`, etc block.